tx consensus parameters

leviathanbeak opened the pull request:
closes #362 

Created `ConsensusParameters` struct with all the necessary fields, and made it part of the `ChainInfo` query - that is, gave it its own queryable method `consensus_parameters`.

adlerjohn reviewed (commented):
looks fairly straightforward

Voxelot commented:
> Would unit tests be necessary as this looks like it's adding a new field on an existing endpoint? Otherwise it looks fairly straightforward and can approve if tests are not needed

Agree, even though this API is very simple and basic, we should have at least one happy path test per public interface.

leviathanbeak commented:
> need to remove unused snapshot, otherwise looks good

ah kept that by accident while rebasing since I started off from where I left the previous PR

leviathanbeak commented:
@Voxelot @ControlCplusControlV adding a e2e test was actually good idea, since there wasn't `chain_info` method on the client at all. I have added it and added a test.

AlicanC reviewed (commented):
LGTM
